### ISSUE TITLE
onOpen is calling when we tap the row

### ISSUE THREAD

**Issue by manjeets12** [dancormier/react-native-swipeout#223]
either we need to update README to mention that it is onPress instead of onOpen, because it is showing behaviour of press, onOpen should be different function that should only be called when row is swiped to some extent or fully swiped

**Comment by munkhorgil** [comment]
Yes exactly, When I try to disable swipeout on open instead it disabled by click.
